modules/acquia_cms_common/src/Services/AcmsUtilityService.php

```diff
@@ -228,6 +228,7 @@ public function getStarterKit(): ?string {
       'acquia_cms_enterprise_low_code' => 'Acquia CMS Enterprise Low Code',
       'acquia_cms_headless' => 'Acquia CMS Headless' ,
       'acquia_cms_community' => 'Acquia CMS Community',
+      'acquia_cms_existing_site' => 'Acquia CMS Existing Site',
     ];
 
     // Check for the starter kit selection.
```

Comment:
please update below if condition with:
```suggestion
$starter_kit = $this->configFactory->get('acquia_cms_common.settings')
    ->get('starter_kit_name');
```